src/app/team/social-sharing/social-sharing.component.html

```diff
@@ -0,0 +1,9 @@
+<a *ngIf="!!githubUrl" mat-button [href]="githubUrl">
+  <mat-icon [ngStyle]="{'color': '#6f066f'}" fontSet="fa" fontIcon="fa-github"></mat-icon>
+</a>
+<a *ngIf="!!twitterUrl" mat-button [href]="twitterUrl">
+  <mat-icon [ngStyle]="{'color': '#4073d4'}" fontSet="fa" fontIcon="fa-twitter"></mat-icon>
+</a>
+<a *ngIf="!!linkedinUrl" mat-button [href]="linkedinUrl">
+  <mat-icon [ngStyle]="{'color': '#4288b3'}" fontSet="fa" fontIcon="fa-linkedin"></mat-icon>
```

Comment:
The Other icons colors have a high contrast. The LinkedIn icon, however, is too light. Can we darken it to #3D7CA4?

Reply:
Fixed in e889fac50e862b496682729ab7c8280768e3030b